config/module.config.php

```diff
@@ -79,117 +79,117 @@
-            'orm_default' => array(),
-        ),
+            'orm_default' => [],
+        ],
 
         // form annotation builder configuration
-        'formannotationbuilder' => array(
+        'formannotationbuilder' => [
             // Configuration for service `doctrine.formannotationbuilder.orm_default` service
-            'orm_default' => array(),
-        ),
+            'orm_default' => [],
+        ],
 
         // entity resolver configuration, allows mapping associations to interfaces
-        'entity_resolver' => array(
+        'entity_resolver' => [
             // configuration for the `doctrine.entity_resolver.orm_default` service
-            'orm_default' => array()
-        ),
+            'orm_default' => [],
+        ],
 
         // authentication service configuration
-        'authentication' => array(
+        'authentication' => [
             // configuration for the `doctrine.authentication.orm_default` authentication service
-            'orm_default' => array(
+            'orm_default' => [
                 // name of the object manager to use. By default, the EntityManager is used
                 'objectManager' => 'doctrine.entitymanager.orm_default',
                 //'identityClass' => 'Application\Model\User',
                 //'identityProperty' => 'username',
-                //'credentialProperty' => 'password'
-            ),
-        ),
+                //'credentialProperty' => 'password',
+            ],
+        ],
 
         // migrations configuration
-        'migrations_configuration' => array(
-            'orm_default' => array(
+        'migrations_configuration' => [
+            'orm_default' => [
                 'directory' => 'data/DoctrineORMModule/Migrations',
                 'name'      => 'Doctrine Database Migrations',
                 'namespace' => 'DoctrineORMModule\Migrations',
                 'table'     => 'migrations',
                 'column'    => 'version',
-            ),
-        ),
+            ],
+        ],
 
         // migrations commands base config
-        'migrations_cmd' => array(
-            'generate' => array(),
-            'execute'  => array(),
-            'migrate'  => array(),
-            'status'   => array(),
-            'version'  => array(),
-            'diff'     => array(),
-            'latest'   => array()
-        ),
-    ),
-
-    'service_manager' => array(
-        'factories' => array(
+        'migrations_cmd' => [
+            'generate' => [],
+            'execute'  => [],
+            'migrate'  => [],
+            'status'   => [],
+            'version'  => [],
+            'diff'     => [],
+            'latest'   => [],
+        ],
+    ],
+
+    'service_manager' => [
+        'factories' => [
             'Doctrine\ORM\EntityManager' => 'DoctrineORMModule\Service\EntityManagerAliasCompatFactory',
-        ),
-        'invokables' => array(
+        ],
+        'invokables' => [
```

Comment:
maybe remove this?
https://zendframework.github.io/zend-servicemanager/migration/#invokables